### ISSUE TITLE
일정 조율은 어떻게 할까요?

### ISSUE THREAD

**Issue by yoonjin09** [yoonjin09/lab-13-mini-project#4]
오늘 수업에서 매일 commit 하나 이상씩은 남기라고 하셔서 그런데 저희 함수 만드는 것은 일정 조율을 어떻게 할까요? 
어느정도 되면 wiki에도 정리하라고 하셔서요! 급한건 아니라 카톡 말고 여기다가 남깁니당

**Comment by yoonjin09** [comment]
일요일부터는 차근 차근 하나씩이라도 만드는게 좋을 것 같습니다! 일요일에 카톡으로 남기겠습니다

**Comment by victory-p** [comment]
확인이 늦었네요!ㅠㅠ
토요일에 조금이라도 만들고 연락드리려고 했는데 연락 못했어요!
일요일부터 한두개씩 매일 커밋하는걸로 할까요?

**Comment by yoonjin09** [comment]
네 그러면 wiki에는 두 개씩 정도 만드는 걸로 일정 올려놓을게요! 만약 wiki대로 안되면 그때 그때 수정하기로 해요!

**Comment by victory-p** [comment]
저는 오늘 추가랑 조회 함수 커밋할게요!

**Comment by yoonjin09** [comment]
네 그러면 저는 main함수랑 listmenu 함수 만들어 놓겠습니다!
